app/controllers/provider_orders_controller.rb

```diff
@@ -1,24 +1,33 @@
 # :nodoc:
 class ProviderOrdersController < ManageCmrController
+  include ApplicationHelper
   add_breadcrumb 'Track Orders', :orders_path
 
   def show
+    init_time_tracking_variables
     logger.tagged("#{current_user.urs_uid} #{controller_name}_controller") do
       @provider_order = generate_provider_order(params['id'])
-
       render :show
     end
+  rescue Faraday::Error::TimeoutError
```

Comment:
I'm not sure if the rescue for the `show` and `edit` actions here are necessary. But if we do it here, is there a reason that it's not being done for the `show` and `index` actions in `orders_controller`?

Reply:
I added for show, but index doesn't perform any faraday request so left as is.